### ISSUE TITLE
Plugin WPTouch not working

### ISSUE THREAD

**Issue by atsheng** [bitnami/bitnami-docker-wordpress#290]
**Description**

I installed plugin wptouch for wordpress and activated. it seems not working on docker container.
I tested wptouch on pure wordpress(not on docker), it works fine.
thanks!

**Steps to reproduce the issue:**

1. Install plugin wptouch.
2. activate wptouch.
3. browse wordpress website using mobile device.
4. when navigate to appearance -> customize, it shows "The theme directory "bauhaus" does not exist."

**Describe the results you received:**

it shows blank page

**Describe the results you expected:**

mobile view for wordpress.

**Version**

- Output of `docker version`:

```
Client:
 Version:           19.03.13
 API version:       1.40
 Go version:        go1.13.15
 Git commit:        cd8016b6bc
 Built:             Fri Feb  5 15:56:39 2021
 OS/Arch:           linux/amd64
 Experimental:      false

Server:
 Engine:
  Version:          19.03.13
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.13.15
  Git commit:       bd33bbf
  Built:            Fri Feb  5 15:58:24 2021
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          v1.3.7
  GitCommit:        8fba4e9a7d01810a393d5d25a3621dc101981175
 runc:
  Version:          1.0.0-rc10
  GitCommit:
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683
```

- Output of `docker info`:

```
Client:
 Debug Mode: false

Server:
 Containers: 9
  Running: 6
  Paused: 0
  Stopped: 3
 Images: 72
 Server Version: 19.03.13
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Native Overlay Diff: true
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 8fba4e9a7d01810a393d5d25a3621dc101981175
 runc version:
 init version: fec3683
 Security Options:
  apparmor
  seccomp
   Profile: default
 Kernel Version: 4.15.0-142-generic
 Operating System: Ubuntu Core 18
 OSType: linux
 Architecture: x86_64
 CPUs: 16
 Total Memory: 15.13GiB
 Name: RAKUbuntuSheng
 ID: VHN5:ULQ4:PZ5R:RQ5J:2S34:CSLX:OYRN:FNB4:R75I:2GTD:VYR7:LEOY
 Docker Root Dir: /var/snap/docker/common/var-lib-docker
 Debug Mode: false
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false
```

- Output of `docker-compose version` (if applicable):

```
docker-compose version 1.25.5, build unknown
docker-py version: 4.4.1
CPython version: 3.6.9
OpenSSL version: OpenSSL 1.1.1  11 Sep 2018
```

**Additional environment details (AWS, VirtualBox, Docker for MAC, physical, etc.):**

```
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.5 LTS
Release:	18.04
Codename:	bionic
```

**Comment by rafariossaa** [comment]
Hi,
  Did you have the chance to check the logs and find errors ?
  Could you provide the logs ?

**Comment by atsheng** [comment]
Hi, thanks for the reply.

I did check the logs in docker-compose logs, I just saw some PHP warning, it didn't have any error related to wptouch.
`PHP Warning:  count(): Parameter must be an array or an object that implements Countable in /bitnami/wordpress/wp-content/plugins/wptouch/core/class-array-iterator.php on line 12`

But with blank page, there should be an error somewhere.
I also enable the flag WP_DEBUG in wp-config.php, didn't see any error either...
If there is another way I can debug this kind of problem?
thank you!
